inst/tutorials/tidy-01-causality1/tidy-01-causality1.Rmd

```diff
@@ -0,0 +1,378 @@
+tut_reptitle <- "QSS Tutorial 1: Output Report"
+data(resume, package = "qss")
+```
+
+
+
+## Conceptual Questions
+
+
+```{r concept-quiz, echo=FALSE}
+quiz(
+  caption = "",
+  question(
+    "Suppose a variable is binary, that is, it takes on values of either 0 or 1 (for example, female gender). Which of the following is the same as its sample mean?",
+    answer("the sample median"),
+    answer("the sample proportion of 1s", correct = TRUE),
+    answer("neither of these")
+  ),
+  question(
+    "What kind of value is `FALSE`?",
+    answer("character"),
+    answer("logical", correct = TRUE),
+    answer("binary"),
+    answer("numeric")
+  ),
+  question(
+    "In order to calculate the mean of a variable we have used the `length()` function in the denominator. The `length()` of a vector is equivalent to:",
+    answer("the number of elements", correct = TRUE),
+    answer("the height"),
+    answer("the maximum")
+  ),
+  question(
+    "How are factor variables different from categorical variables?",
+    answer("They are the same", correct = TRUE),
+    answer("Factor variables contain numeric values"),
+    answer("Categorical variables tend to have more levels or categories")
+  )
+)
+```
+
+
+## Working with Logicals in R
+
+### Exploring the resume data
+
+
+In this tutorial, we are going to be working with the resume data from Section 2.1 of QSS. This data comes from an experiment where researchers sent fictitious resumes with different names that implied different race and gender combinations to see if potential employers were more likely to call back names associated with different racial groups and genders.
+
+Let's first explore the data a bit. It's stored as `resume`.
+
+#### Exercise
+
+- Use the `head` function to show the first six lines of the `resume` data.
+
+
+```{r head, exercise=TRUE}
+## print the first 6 lines of the data
+
+```
+
+```{r head-solution}
+head(resume)
+```
+
+```{r head-check}
+grade_code()
+```
+
+- Print out the dimension of the `resume` data.
+
+
+```{r resume-dim, exercise=TRUE}
+
+```
+
+```{r resume-dim-solution}
+dim(resume)
+```
+
+```{r resume-dim-check}
+grade_code()
+```
+
+
+- Use the `summary` function to show a summary of the data.
+
+
+```{r res-summary, exercise=TRUE}
+
+```
+
+```{r res-summary-solution}
+summary(resume)
+```
+
+```{r res-summary-check}
+grade_code()
+```
+
+
+### Creating a cross tab
+
+
+To help you analyze this data, you can use a cross tabulation. Cross tabulation (or contingency table) is a table that quickly summarizes categorical data. For instance, in the resume data, we have a `sex` variable that tells us whether or not the fictitious resume had a male or a female name.
+
+#### Exercise
+
+- Using the `group_by` and `count` and `pivot_wider` functions, create a cross tab of the `sex` and `call` variables in the resume data.
+
+```{r table, exercise = TRUE}
+
+```
+
+```{r table-solution}
+resume %>%
+  group_by(sex, call) %>%
```

Comment:
`resume %>% count(sex, call)` is shorter and does the same thing than group_by + count

Reply:
edited!